index.js

```diff
@@ -170,7 +160,7 @@ function renderMedia (file, getElem, opts, cb) {
     }
 
     function useVideostream () {
-      debug('Use `videostream` package for ' + file.name)
+      debug(`Use 'videostream\` package for ${file.name}`)
```

Comment:
I think you replaced a backtick with a single quote here. To output the same string you'd need 
```js
`Use \`videostream\` package...`
```